CMakeLists.txt

```diff
@@ -52,8 +51,7 @@ if(OPENSN_WITH_PYTHON_MODULE)
 endif()
 
 if(OPENSN_WITH_LUA)
-    set(OPENSN_WITH_PYTHON OFF CACHE BOOL "Lua support requested, disabling Python interface" FORCE)
-    set(OPENSN_WITH_PYTHON_MODULE OFF CACHE BOOL "Lua support requested, disabling Python module" FORCE)
+  message(FATAL_ERROR "The option OPENSN_WITH_LUA has been removed and is no longer supported.")
 endif()
```

Comment:
I don't think you want to keep this. The `OPENSN_WITH_LUA` has been removed, so all the if-statements with `OPENSN_WITH_LUA` should be removed as well. We have the advantage that there was no release yet, so you don't have to go through the deprecation circle and you can just remove stuff.